airbyte-integrations/connectors/destination-pinecone/Dockerfile

```diff
@@ -38,6 +38,6 @@ COPY destination_pinecone ./destination_pinecone
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.0.4
+LABEL io.airbyte.version=0.0.5
```

Comment:
A nit, but this is a significant enough refactor that I’d personally go with a minor version bump. That would bring us to `0.1.0`. I’ll defer to your call though on this point, since it can certainly be argued wither way.

Reply:
As it's not changing behavior I don't think we need to make a big bump